.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
```

Comment:
`Bug` in https://github.com/bitcoin/bitcoin/labels

Reply:
Good catch